2308.03843/2308.03843.yaml

```diff
@@ -10,56 +10,58 @@
       score: 
         Haining: 2
         Will: 
-        Yasaman: 
+        Yasaman: 2
     real|momentum:
       LLM: real
       human: 
       score: 
         Haining: 2
         Will: 
-        Yasaman: 
+        Yasaman: 2
     single-particle|second-quantized:
       LLM: second-quantized
       human: 
       score: 
         Haining: 2
         Will: 
-        Yasaman: 
+        Yasaman: 2
     degrees_of_freedom:
       LLM:  spin and sublattice
       human: 
       score: 
         Haining: 2
         Will: 
-        Yasaman: 
+        Yasaman: 1
+        # partial credit -- missing unit cell
```

Comment:
Yes, but that's because I delibrately want to separate "real/momentum" index from those "discretized index" like orbital and spin.